includes/validation/class-amp-validation-manager.php

```diff
@@ -898,7 +897,7 @@ public static function print_edit_form_validation_status( $post ) {
 				esc_html_e( 'However, your site is configured to automatically accept sanitization of the offending markup. You should review the issues to confirm whether or not sanitization should be accepted or rejected.', 'amp' );
 			} else {
 				/*
-				 * Even if the 'auto_accept_sanitization' option is true, if there are non-accepted errors in non-Standard mode, it will redirect to a non-AMP page.
+				 * Even if sanitizations are auto accepted, if there are non-accepted errors in non-Standard mode, it will redirect to a non-AMP page.
```

Comment:
```suggestion
				 * Even if sanitizations are auto-accepted, if there are non-accepted errors in non-Standard mode, it will redirect to a non-AMP page.
```